packages/backend-modules/translate/translations.json

```diff
@@ -452,6 +452,14 @@
       "key": "api/email/option/goodie/notebook/other",
       "value": "Notizbücher"
     },
+    {
+      "key": "api/email/option/goodie/librarybook/1",
+      "value": "Republik-Bibliothek Taschenbuch"
+    },
+    {
+      "key": "api/email/option/goodie/librarybook/other",
```

Comment:
I'm a bit unsure about the lowercase keys here, compared to elsewhere but I guess it's correct that email variables are lowercased for some reason 😅